packages/ui/src/FocusAreaCard.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image'
+
+import { Heading } from '@filecoin-foundation/ui/Heading'
+import { Icon, type IconProps } from '@filecoin-foundation/ui/Icon'
+import type { StaticImageProps } from '@filecoin-foundation/utils/types/imageType'
+
+type FocusAreaImageProps = StaticImageProps & {
+  sizes?: string
+}
+
+export type FocusAreaCardProps = {
+  title: string
+  children: React.ReactNode
+  icon?: IconProps['component']
+  image?: FocusAreaImageProps
+}
+
+export function FocusAreaCard({
+  title,
+  children,
+  image,
+  icon,
+}: FocusAreaCardProps) {
+  return (
+    <li className="focus-area-card">
+      {image && <FocusAreaCard.Image image={image} />}
+      {icon && <FocusAreaCard.Icon icon={icon} />}
+      <div className="focus-area-card-content">
+        <Heading tag="h3" variant="lg">
+          {title}
+        </Heading>
+        {children}
+      </div>
+    </li>
+  )
+}
+
+FocusAreaCard.Image = function FocusAreaCardImage({
+  image,
+}: Pick<FocusAreaCardProps, 'image'>) {
```

Comment:
At this point, this is not the correct type anymore. Image should be present.

```suggestion
FocusAreaCard.Image = function FocusAreaCardImage({
  image,
}: {
  image: NonNullable<FocusAreaCardProps['image']>
}) {
```